namui/src/namui/system/typeface/load_sans_typeface_of_all_languages.rs

```diff
@@ -26,32 +26,52 @@ async fn load_fallback_font_typefaces() -> Result<(), Box<dyn std::error::Error>
 
 async fn get_noto_color_emoji_typeface() -> Result<Typeface, Box<dyn std::error::Error>> {
     let url = crate::Url::parse("bundle:__system__/font/NotoColorEmoji.woff2")?;
-    let bytes = crate::file::bundle::read(&url)
-        .await
-        .expect(format!("Could not fetch {}", url).as_str());
-
-    Ok(Typeface::new(&bytes))
+    get_typeface(url).await
```

Comment:
just, rollback this changes and make a change to the line of data fetch.

```rust
// before
let bytes = crate::file::bundle::read(&url).await

// after
let bytes = get_data_from_cache_first_then_bundle_blah_blah_some_good_name(&url).await
```

Then everything is same. You don't have to changes the structure of codes.